pkg/cluster/context.go

```diff
@@ -178,6 +178,18 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// try to remove the kind kube config file generated by "kind create cluster"
+	err = os.Remove(c.KubeConfigPath())
+	if err != nil {
+		log.Warningf("Tried to remove %s but recieved error: %s\n", c.KubeConfigPath(), err)
+	}
+
+	// check if $KUBECONFIG is set and let the user know to unset if so
+	if os.Getenv("KUBECONFIG") == c.KubeConfigPath() {
```

Comment:
note to self: a more complete version of this will need to consider how to handle multiple files, which KUBECONFIG / kubectl support.